support-frontend/assets/pages/[countryGroupId]/guardianLightLanding/components/guardianLightCard.tsx

```diff
@@ -0,0 +1,221 @@
+import { css, ThemeProvider } from '@emotion/react';
+import {
+	from,
+	headlineBold17,
+	headlineBold24,
+	palette,
+	space,
+	until,
+} from '@guardian/source/foundations';
+import { LinkButton, themeLinkBrand } from '@guardian/source/react-components';
+import { Divider } from '@guardian/source-development-kitchen/react-components';
+import { BenefitsCheckList } from 'components/checkoutBenefits/benefitsCheckList';
+import { type ProductDescription } from 'helpers/productCatalog';
+
+export type CardPosition = 1 | 2;
```

Comment:
If we start CardPosition from 0 eg. `CardPosition = 0 | 1`, then you won't need the`cardIndexToTier` mapping here: https://github.com/guardian/support-frontend/pull/6577/files#diff-3192b33b7ca9c8a227de6393ed299b941141d8ae77532f38e95464d7e7f07a06R31

Reply:
Yes, less code and simpler, have used 0,1 over 1,2